db/access.php

```diff
@@ -35,13 +35,5 @@
                 'archetypes'   => array(
                         'manager' => CAP_ALLOW
                 )
-            ),
-        'tool/logstorexapi:manageerrors' => array(
-                'riskbitmask'  => RISK_CONFIG,
-                'captype'      => 'read',
-                'contextlevel' => CONTEXT_SYSTEM,
-                'archetypes'   => array(
-                      'manager' => CAP_ALLOW
-                )
```

Comment:
why has this been removed?

Reply:
That should still be present. I will add it back.